### PR TITLE
Setup tests.

### PR DESCRIPTION
**Pull request by martinzlocha**
Currently set up to pass if accuracy is above 45%.

### CODE REVIEW

File: WLC/tests.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from unittest import TestCase
```
Comment (mishrabhinav) [inline WLC/tests.py]: You don't really need to use `unittest` if you are running the tests with `pytest`.

File: WLC/tests.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+from WLC.benchmark import run_benchmarks
+
+MINIMUM_ACCURACY = 45
+
+
+class BenchmarkTest(TestCase):
+    def testBenchmarks(self):
```
Comment (mishrabhinav) [inline WLC/tests.py]: Convert this to snake_case

File: WLC/benchmark.py [hunk 3]
```diff
@@ -27,18 +27,34 @@ def benchmark_file(file_name):
...
+
     for i in range(1, 6):
-        benchmark_file('assets/examples/images/example_{}.png'.format(i))
+        accuracy, length = benchmark_file('assets/examples/images/example_{}.png'.format(i))
+        total_accuracy += accuracy * length
+        total_length += length
+
+    overall_accuracy = round(total_accuracy / total_length)
+
+    LOGGER.info('')
```
Comment (mishrabhinav) [inline WLC/benchmark.py:53]: I know it's for good formatting but it doesn't look nice.

File: WLC/tests.py (hunk 1 above)
Comment (martinzlocha) [reply WLC/tests.py]: Fixed

File: WLC/tests.py (hunk 2 above)
Comment (martinzlocha) [reply WLC/tests.py]: Fixed

File: WLC/benchmark.py (hunk 3 above)
Comment (martinzlocha) [reply WLC/benchmark.py:53]: Its not the nicest but will keep for formatting.

**Pull request merged**